addr_manager_test.go

```diff
@@ -1,9 +1,14 @@
 package peerstore
 
 import (
+	"context"
+	"io/ioutil"
+	"os"
 	"testing"
 	"time"
 
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-ds-badger"
```

Comment:
some leftover badger stuff here

Reply:
actually still need this for creating the test datastores

Reply:
You can use memory datastore.